frame/election-provider-support/solution-type/tests/ui/fail/wrong_attribute.rs

```diff
@@ -5,6 +5,7 @@ generate_solution_type!(
 		VoterIndex = u8,
 		TargetIndex = u16,
 		Accuracy = Perbill,
+		MaxVoters = frame_support::traits::ConstU32::<10>,
```

Comment:
Probably you don't even need to have a proper import here, right? I mean, Perbill is not in scope and it works? because we fail before actually compiling anything.